src/components/Toast/Toast.tsx

```diff
@@ -1,91 +1,75 @@
+/* TODO: point to internal exports once fully migrated */
+// import { Text, CloseButton } from '@chanzuckerberg/eds';
+// import colorStyles from '@chanzuckerberg/eds/lib/common/Notifications/Notification.module.css';
+// import NotificationIcon from '@chanzuckerberg/eds/lib/common/Notifications/NotificationIcon';
+
 import clsx from 'clsx';
-import React, { ReactNode, useState } from 'react';
+import React from 'react';
 import styles from './Toast.module.css';
-import { Button } from '../Button/Button';
-import { Icon } from '../Icon/Icon';
 
-export interface Props {
-  /**
-   * The child node(s) contains the toast message. Note: the toast message is displayed inside a TextPassage, so children can contain raw HTML
-   */
-  children?: ReactNode;
+export type Variant = 'success' | 'alert';
+
+export type Props = {
   /**
-   * CSS class names that can be appended to the component.
```

Comment:
when possible, it'd be good to integrate any existing comments for props. esp. b/c sometimes they're more verbose / descriptive 🙂

Reply:
Will merge comments